Source/pages/repo_manage_page.php

```diff
@@ -121,8 +122,12 @@ function convert_to_key_value( $p_array ) {
 			foreach( $t_formatted_array as $t_key => $t_value ) {
 		?>
 		<tr>
-			<td class="category"><?php echo $t_key ?></td>
+			<td class="category"><?php echo plugin_lang_get( $t_key, $t_vcs->basename ) ?></td>
+<?php if( is_bool( $t_value ) ) { ?>
+			<td><?php echo trans_bool( $t_value ) ?></td>
+<?php } else { ?>
 			<td><?php echo $t_value ?></td>
+<?php } ?>
```

Comment:
To make the code easier to read, I would suggest to use a ternary operator here, i.e.
``` php
<td><?php echo is_bool( $t_value ) ? trans_bool( $t_value ) : $t_value ?></td>
```